tests/client-dbus/check.py

```diff
@@ -5,23 +5,19 @@
 import sys
 
 arg_map = {
-   "src/stratisd_client_dbus" : [
-      "--reports=no",
-      "--disable=I",
-      "--disable=bad-continuation",
-      "--disable=invalid-name",
-      "--msg-template='{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}'"
-   ],
-   "tests" : [
-      "--reports=no",
-      "--disable=I",
-      "--disable=bad-continuation",
-      "--disable=duplicate-code",
-      "--disable=invalid-name",
-      "--msg-template='{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}'"
-   ]
```

Comment:
I think the original formatting is better.  We could add: "# yapf: disable" and yapf will leave it alone.